src/lib/pay-request/typed_clients/client.ts

```diff
@@ -30,7 +30,7 @@ export function config(env: NodeJS.ProcessEnv = {}, options: PayHooks = {}): voi
   const adminUsersUrl = env.ADMINUSERS_URL
   const ledgerUrl = env.LEDGER_URL
   const productsUrl = env.PRODUCTS_URL
-  const publicAuthUrl = env.PUBLICAUTH_URL
+  const publicAuthUrl = env.PUBLIC_AUTH_URL
```

Comment:
Does the environment actually have this property? Might need a small tf change alongside this

Reply:
Yep, the new variable matches up with the variable in the terraform